src/openrct2/drawing/TTFSDLPort.cpp

```diff
@@ -403,7 +403,7 @@ static TTF_Font* TTF_OpenFontIndexRW(FILE* src, int freesrc, int ptsize, long in
     font->src = src;
     font->freesrc = freesrc;
 
-    stream = static_cast<FT_Stream>(malloc(sizeof(*stream)));
+    stream = new FT_StreamRec_;
```

Comment:
unique_ptr here as well.

Reply:
I was considering that too, but:
-  `stream` is part of the `font` object so similar ownership issue as above
-  all variables in this function are C-style declared at the beginning of the function so I left it the for consistency as well

Reply:
See comment above.

Reply:
I replied above as well.